tests/ArrayTrait.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace WP_Rocket\Tests;
+
+trait ArrayTrait {
```

Comment:
@Tabrisrp This trait gives us some handy array functionality for working with the filesystem, config, etc. Do you recognize the intent of this code? It’s inspired by older versions of Laravel.